solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -598,6 +598,6 @@ pub enum EntityTypes {
 // Seed & bump used to validate the user's handle with the account base
 #[derive(AnchorSerialize, AnchorDeserialize, Clone, PartialEq)]
 pub struct UserHandle {
```

Comment:
We should stick to using this struct but I can make this change later as I propagate validation everywhere.